whylog/converters/exceptions.py

```diff
@@ -0,0 +1,9 @@
+from whylog.exceptions import WhylogError
+
+
+class UnsupportedConverter(WhylogError):
+    def __init__(self, converter_class_name):
+        self.converter_class_name = converter_class_name
+
+    def __str__(self):
+        return 'This whylog version do not handle %s. Please upgrade Whylog' % self.converter_class_name
```

Comment:
It could be both of your cases. Whylog developer must fix broken code, but normal Whylog user must download the newest version. So ```Please upgrade Whylog ``` left